Collection Classes/Vector Class/int indexOf(Object elem, int index)/int indexOf(Object elem, int index).java

```diff
@@ -0,0 +1,52 @@
+/*
+ int indexOf(Object elem, int index): This method returns the index of the first occurrence of 
+ the specified element in this vector, or -1 if this vector does not contain the element after the given index position.
+
+*/
+
+import java.util.*;
+
+class VectorIndexOf {
+	
+    //main code
+    public static void main ( String[ ] args )
+    {
+	Scanner sc=new Scanner ( System.in );
+	    
+	//n is the size of vector
+	int n = sc.nextInt();
+	    
+        // Declare an empty vector
+         Vector<String> vc = new Vector<>(); 
+         
+         Vector<String> vc1 = new Vector<>(); 
```

Comment:
need of vc1? @plazzy99

Reply:
ohh i just used the previous code as template of this one so by mistake